OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -299,12 +294,11 @@ public static User ToDomain(this ShortUserDto shortUserDto, User user)
             });
         }
 
-        public static Workshop ToDomain(this WorkshopDTO workshopDto)
+        public static Workshop ToDomain(this WorkshopCard workshopCard)
```

Comment:
Why need to change this method? The DTO model is used for creating and updating. As I understand, the task was only for GET-requests for Applications to receive short model.